charts/accurate/values.yaml

```diff
@@ -102,3 +102,8 @@ controller:
           - update
           - patch
           - delete
+    # controller.additionalRBAC.clusterRoles -- Specify additional ClusterRoles to be granted
+    # to the accurate controller. "admin" is recommended to allow the controller to manage
+    # common namespace-scoped resources.
+    clusterRoles:
+    - admin
```

Comment:
```suggestion
      - admin
```

Nit

Reply:
Thanks. But I don't take formatting preference unless we enforce it by checking it on CI.

Reply:
I just think we should have consistent formatting in the same file. 😉